src/Components/RosTable/RenderColumn.js

```diff
@@ -14,28 +15,35 @@ export const diskUsageData = (data, id, item) => {
         state === NO_DATA_STATE ?
             <span>{ NO_DATA_VALUE }</span> :
             <Tooltip position="right" content={
-                <div>
-                    <table>
-                        <tr>
-                            <th>Device name</th>
-                            <th>Value</th>
-                        </tr>
-                        <tr>
-                            <td colSpan="100%" className="seperator"></td>
-                        </tr>
+                <TableComposable
+                    arial-label="disk usage"
+                    variant="compactBorderless"
+                    borders={false}
+                >
+                    <Thead>
+                        <Tr>
+                            <Th modifier="nowrap" textCenter style={{
```

Comment:
Instead of applying inline style & that is at multiple places, will it make sense to create CSS class and pass it using `className`?

Reply:
@kgaikwad I have already tried it. It is not working here. I have asked @PreetiW to take a look

Reply:
@kgaikwad I tried to use below class on the `table` component here but it is not working, Do you have any suggestions on this?

`.disk-table.pf-c-table {
    --pf-global--BackgroundColor--100: var(--pf-global--palette--black-1000);
    --pf-global--Color--100: var(--pf-global--BackgroundColor--100);
    --pf-c-table--BackgroundColor: var(--pf-global--palette--black-1000);
}`